crates/bevy_ecs/src/world/entity_ref.rs

```diff
@@ -272,6 +274,14 @@ impl<'w> EntityRef<'w> {
         // SAFETY: We have read-only access to all components of this entity.
         unsafe { self.0.get_components::<Q>() }
     }
+
+    /// Returns the source code location from which this entity has last been spawned
+    /// or despawned. Returns `None` if this entity has never existed.
+    /// In a removal hook or observer, this points to the despawn.
+    #[cfg(feature = "track_change_detection")]
+    pub fn get_spawned_despawned_by(&self) -> &'static Location<'static> {
+        self.0.get_spawned_despawned_by()
+    }
```

Comment:
I believe this documentation is out-of-date, since this function does not return an `Option`.